app/AttributionGenerator.js

```diff
@@ -158,7 +158,9 @@ $.extend( AttributionGenerator.prototype, {
 		// Prefer attribution over author:
 		var $attribution = this._asset.getAttribution();
 		if( $attribution ) {
-			return format === 'html' ? $attribution : $attribution.text();
+			if( $attribution.length && $attribution.text().length ) {
```

Comment:
In the WB coding style guidelines we have something about not doing implicit conversions like the cast from `$attribution.text().length` to boolean. `$attribution.text().length > 0`